linetools/isgm/abscomponent.py

```diff
@@ -167,6 +174,167 @@ def build_table(self):
         # Return
         return comp_tbl
 
+    def cog(self, redo_EW=False, show_plot=False, **kwargs):
+        """Perform a COG analysis on the component
+
+        Parameters
+        ----------
+        redo_EW : bool, optional
+          Re-analyze each line for its EW
+        show_plot : bool, optional
+          Generate plot and show
+
+        Returns
+        -------
+        logN : float
+          COG column density
+        b : Quantity
+          COG Doppler parameter (km/s)
+        """
+        from linetools.analysis import cog as ltcog
+        reload(ltcog)
+        # Redo EWs?
+        if redo_EW:
+            for aline in self._abslines:
+                aline.measure_restew(**kwargs)
+        # COG setup
+        wrest = np.array([aline.wrest.to('AA').value for aline in self._abslines])*u.AA
+        f = np.array([aline.data['f'] for aline in self._abslines])
+        EW = np.array([aline.attrib['EW'].to('AA').value for aline in self._abslines])*u.AA
+        sigEW=np.array([aline.attrib['sigEW'].to('AA').value for aline in self._abslines])*u.AA
+        # COG analysis
+        COG_dict = ltcog.single_cog_analysis(wrest, f, EW, sigEW=sigEW)
+        # COG plot
+        if show_plot:
+            ltcog.cog_plot(COG_dict)
+        # Return
+        return COG_dict
+
+
+    def plot_Na(self, show=True, **kwargs):
+        """Plot apparent column density Na profiles
+        """
+        import matplotlib.pyplot as plt
+        import matplotlib.gridspec as gridspec
+        import matplotlib as mpl
+        try: # Nicer view, especially in notebook
+            import seaborn as sns; sns.set(context="notebook",font_scale=2)
+        except:
```

Comment:
maybe change to `except ImportError:`. Catching all exceptions is seen as bad style among Python people.